hdtv/plugins/calInterface.py

```diff
@@ -794,8 +855,15 @@ def CalFromFile(self, fname):
 
         return hdtv.cal.MakeCalibration(calpoly)
 
-    def CalFromPairs(self, pairs, degree=1, table=False,
-                     fit=False, residual=False, ignore_errors=False):
+    def CalFromPairs(
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)